extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsTopologyManager.java

```diff
@@ -103,22 +100,17 @@ void onStart(@Observes StartupEvent ev) {
             return;
         }
 
+        runtimeConfig.topics = runtimeConfig.topics.stream().map(String::trim).collect(Collectors.toList());
```

Comment:
I'm not excited about this: I don't think we should change the config value.

Can you use a local variable instead?

Thanks!

Reply:
I refactored this piece of code.

I added the `getTrimedTopics()` method in the `KafkaStreamsRuntimeConfig` class to avoid futur code duplication about triming on `topics`.